lib/paper_trail/events/update.rb

```diff
@@ -25,7 +25,8 @@ def initialize(record, in_after_callback, is_touch, force_changes)
       # @api private
       def data
         data = {
-          item: @record,
+          item_id: record_id,
+          item_type: @record.class.base_class.name,
```

Comment:
Calculation of `item_type` is duplicated here and in `create.rb`. Should we fix this duplication by extracting a method into the parent class?

Reply:
sounds good